### ISSUE TITLE
Progress bar breaking on solr indexing when count of pages is off

### ISSUE THREAD

**Issue by quadrismegistus** [Princeton-CDH/ppa-django#482]

**Comment by quadrismegistus** [comment]
Going to just fix this by setting max_error to be False on pbar.ProgressBar:

```
class ProgressBar(StdRedirectMixin, ResizableMixin, ProgressBarBase):
    '''The ProgressBar class which updates and prints the bar.

    Args:
        min_value (int): The minimum/start value for the progress bar
        max_value (int): The maximum/end value for the progress bar.
                            Defaults to `_DEFAULT_MAXVAL`
...
        max_error (bool): When True the progressbar will raise an error if it
            goes beyond it's set max_value. Otherwise the max_value is simply
            raised when needed
```

**Comment by rlskoeser** [comment]
Tested indexing pages yesterday multiple times on the new test vm server, and never got the page count error we saw when we first ran it after the pgloader migration.

I also experimented a little bit with the number of processes. Our test VM has 2 core, and the script uses the same number of processes as cores by default — when I ran with 4 processes instead of 2 it was a bit faster. I _suspect_ this is because indexing Gale content requires waiting on their API, which can be slow for larger texts.

Here is the `time` output for a couple of different runs:

Indexing with default 2 processes:
```
Items in Solr by item type: 1897884 pages, 6456 works

real    95m13.133s
user    45m49.996s
sys     0m28.560s
```

Indexing with 4 processes:
```
Items in Solr by item type: 1897884 pages, 6456 works

real    50m1.842
user    53m15.275s
sys     0m42.020s
```
I tested with 8 processes just to see if it made any difference, and it was about the same as the time for 4 processes.
